src/sentry/models/integrations/sentry_app_installation.py

```diff
@@ -39,22 +42,20 @@ def get_installed_for_organization(self, organization_id: int) -> QuerySet:
     def get_by_api_token(self, token_id: str) -> QuerySet:
         return self.filter(status=SentryAppInstallationStatus.INSTALLED, api_token_id=token_id)
 
-    def get_projects(self, token: ApiToken) -> QuerySet[Project]:
+    def get_projects(self, token: ApiToken | AuthenticatedToken) -> QuerySet[Project]:
         from sentry.models import Project, SentryAppInstallationToken
 
-        try:
-            installation = self.get_by_api_token(token.id).get()
-        except SentryAppInstallation.DoesNotExist:
-            installation = None
+        token_id = token.id if isinstance(token, ApiToken) else token.entity_id
```

Comment:
I created a helper for this in the future, when mixing tokens with authenticated tokens.  Next commit.